apps/server/lib/lexical/server/code_intelligence/completion/translations/module_or_behaviour.ex

```diff
@@ -124,7 +124,7 @@ defmodule Lexical.Server.CodeIntelligence.Completion.Translations.ModuleOrBehavi
 
     local_name =
       child_module
-      |> String.slice(local_module_length..-1)
+      |> String.slice(local_module_length..-1//1)
```

Comment:
Did range behaviour change in 1.16?

Reply:
yes.

> [Enum] Deprecate passing a range with negative step on Enum.slice/2, give first..last//1 instead

Reply:
wow, that's a usability regression. Bummer.